docs/userGuide/syntax/lists.md

```diff
@@ -91,9 +91,20 @@ If either the `i-width` or the `i-height` of an image is not specified, the unsp
 </variable>
 </include>
   
+**The spacing between the icon and the content can be customized by using a `i-spacing` attribute.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* Item 1 { icon="+1" }
+* Item 2 { i-spacing="1rem" }
+* Item 3 { i-spacing="2rem" }
+</variable>
+</include>
+
 <box type=tip seamless>
 
-Similar to the `icon` attribute, other icon attributes such as `i-class`, `i-width`, `i-height` apply for subsequent list items at the same level, until they are overridden by the same attribute. For example, Item 2.3's `i-class` overrides Item 2.1's and applies up to Item 3.1.
+Similar to the `icon` attribute, other icon attributes such as `i-class`, `i-width`, `i-height`, `i-spacing` apply for subsequent list items at the same level, until they are overridden by the same attribute. For example, Item 2.3's `i-class` overrides Item 2.1's and applies up to Item 3.1.
```

Comment:
This was the comment that this docs was missing. I guess we should merge this first....

Reply:
Could this be placed at a better spot? This is kinda in the middle. (So, users may miss it)

Reply:
I think this is okay...

Reply:
Ill add another comment in the other PR earlier regarding this cascading effect!